### PR TITLE
Add support for http-interop/http-middleware 0.5

### PR DESCRIPTION
**Pull request by danizord**
The proposed PSR-15 interfaces changed again breaking BC, but @webimpress made this magic to support it without breaking BC :D

### CODE REVIEW

**Comment by bakura10** [comment]
Can you please give a few details on what are the breaks ? :p

**Comment by danizord** [comment]
@bakura10 the PSR-15 proposal breaks BC by renaming `DelegateInterface` to `RequestHandlerInterface` and its `process()` method to `handle()` (see https://github.com/php-fig/fig-standards/pull/926).

But here in ZfrEbWorker we don't break BC at all, thanks to the https://github.com/webimpress/http-middleware-compatibility compatibility layer.

File: test/Listener/SilentFailingListenerTest.php [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 namespace ZfrEbWorkerTest\Listener;
 
-use Interop\Http\ServerMiddleware\DelegateInterface;
 use Psr\Http\Message\ServerRequestInterface;
+use Webimpress\HttpMiddlewareCompatibility\HandlerInterface as DelegateInterface;
+use Webimpress\HttpMiddlewareCompatibility\MiddlewareInterface;
```
Comment (michalbundyra) [inline test/Listener/SilentFailingListenerTest.php]: Above line is not used. Can be removed.

**Review by michalbundyra — APPROVED** [review]
LGTM 👍

**Comment by bakura10** [comment]
Looks good. Can you please just add this in the changelog as 6.1.0 ?

**Comment by bakura10** [comment]
Tagged as 6.1.0 !

**Pull request merged**